zerver/management/commands/compilemessages.py

```diff
@@ -111,6 +111,30 @@ def extract_language_options(self) -> None:
                     'locale': 'en',
                 })
                 continue
+            if locale == 'hi':
+                data['languages'].append({
+                    'name': 'Hindi',
+                    'name_local': 'हिंदी',
+                    'code': 'hi',
+                    'locale': 'hi',
+                })
+                continue
+            if locale == 'zh_TW':
+                data['languages'].append({
+                    'name': 'chinese',
+                    'name_local': '中文',
+                    'code': 'zh-tw',
+                    'locale': 'zh_TW',
+                })
+                continue
+            if locale == 'ml':
+                data['languages'].append({
+                    'name': 'Malayalam',
+                    'name_local': 'മലയാളം',
```

Comment:
I am a bit confused why Django is not storing `name_locale` as മലയാളം instead of Malayalam in first place.

https://github.com/django/django/blob/master/django/conf/locale/__init__.py#L375

Reply:
I changes it my self  , because I thought that writing it like this will not affect anything . so I will change it .

Reply:
@ritikchauhan-01 No. No need to revert the change. I was just curious why Django upstream doesn't have the name_locale correct.

Reply:
okay